ush/python/pygfs/task/analysis.py

```diff
@@ -0,0 +1,71 @@
+from pygw.task import Task
+from pygw.attrdict import AttrDict
+from pygw.template import Template, TemplateConstants
+from pygw.yaml_file import YAMLFile
+from pygw.file_utils import FileHandler
+import datetime as dt
+import logging
+import os
+
+crtmver = '2.3.0'
```

Comment:
Shouldn't we be getting this from the environment? The CRTM version is already being set in `parm/config/config.aeroanl`.

Reply:
Yeah this is a dumb leftover that needs removed, thanks for the catch.